src/components/body/components/sideBar/components/teamMateList/components/teamMateDetail/TeamMateDetail.tsx

```diff
@@ -0,0 +1,49 @@
+//libs
+import { useCallback } from "react";
+
+//component
+import { Avatar } from "../../../../../../../avatar/Avatar";
+
+//hooks
+import { useUserId } from "../../../../../../../../hooks/UserContext";
+
+//type
+import { User } from "../../../../../../../../types/types";
+
+//style
+import styles from "./TeamMateDetail.module.css";
+type TeamMateDetailProps = {
+  teamMate: User;
+  onChangeActiveTeamMate: (activeTeamMateId: User) => void;
+  isActive: boolean;
+};
+
+export const TeamMateDetail = ({
+  teamMate,
+  onChangeActiveTeamMate,
+  isActive,
+}: TeamMateDetailProps) => {
+  const handleClick = useCallback(
+    () => onChangeActiveTeamMate(teamMate),
+    [onChangeActiveTeamMate, teamMate]
+  );
+  const highlightClass = isActive ? styles.highlight : "";
+  const userId = useUserId();
+  return (
+    <div
+      className={`${styles.teamMateDetail} ${highlightClass}`}
+      onClick={handleClick}
```

Comment:
We should avoid using clickable divs - [Read this](https://benmyers.dev/blog/clickable-divs/)
Here it's fine as we will need to reset the native styles provided to the button.

For this specific case, we have `BaseButton`.